package.json

```diff
@@ -42,6 +42,7 @@
     "test:unit:login": "BABEL_ENV=test mocha --opts src/platform/testing/unit/mocha.opts --recursive 'test/login/**/*.unit.spec.js?(x)' ",
     "test:unit:personalization": "BABEL_ENV=test mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/applications/personalization/**/tests/**/*.unit.spec.js?(x)' ",
     "test:unit:db": "BABEL_ENV=test mocha --opts src/platform/testing/unit/mocha.opts --recursive 'test/disability-benefits/**/*.unit.spec.js?(x)' ",
+    "test:unit:4142": "BABEL_ENV=test mocha --opts src/platform/testing/unit/mocha.opts --recursive 'src/applications/disability-benefits/4142/**/tests/**/*.unit.spec.js?(x)' ",
```

Comment:
Running this locally, you could also add a `describe.only` test block so you don't have to run all the tests every time when you use `npm run test:unit`. Don't think this warrants adding to master, though.